osu.Game/Screens/Backgrounds/BackgroundModeBeatmap.cs

```diff
@@ -22,17 +22,19 @@ public WorkingBeatmap Beatmap
             {
                 return beatmap;
             }
-
             set
             {
-                if (beatmap == value)
+                if (beatmap == value && beatmap != null)
                     return;
-
                 beatmap = value;
 
                 Schedule(() =>
                 {
-                    Background newBackground = new BeatmapBackground(beatmap);
+                    Background newBackground;
+                    if (beatmap == null)

```

Comment:
What I mean with "whole class of problems" is that _every single time a beatmap is used_ you need a special treatment for the null case. Every. Single. Time. In the old code base those were literally hundreds of times.

What do you think is more annoying than that about having a default beatmap that avoids all this special treatment of a null beatmap?